helpers/fixtures.ts

```diff
@@ -0,0 +1,167 @@
+import {
+  Analysis,
+  AnalysisStrategy,
+  AttributionWindowSeconds,
+  ExperimentFull,
+  MetricAssignment,
+  MetricBare,
+  Platform,
+  RecommendationReason,
+  Status,
+  Variation,
+} from '@/models'
+
+function createAnalysis(fieldOverrides: Partial<Analysis>) {
+  return new Analysis({
+    metricAssignmentId: 123,
+    analysisStrategy: AnalysisStrategy.IttPure,
+    participantStats: {
+      total: 1000,
+      not_final: 100,
+      variation_1: 600,
+      variation_2: 400,
+    },
+    metricEstimates: {
+      diff: { estimate: 0.0, bottom: -0.01, top: 0.01 },
+      variation_1: { estimate: 0.12, bottom: 0, top: 10.0 },
+      variation_2: { estimate: -0.12, bottom: -1.123, top: 1.0 },
+    },
+    recommendation: {
+      endExperiment: true,
+      chosenVariationId: 2,
+      reason: RecommendationReason.CiInRope,
+      warnings: [],
+    },
+    analysisDatetime: new Date(2020, 4, 10),
```

Comment:
If these are being used for tests, we probably want to ensure this value is consistent across timezones.
```suggestion
    analysisDatetime: new Date(Date.UTC(2020, 4, 10)),
```

Note: Comment applies to other dates too.